.github/workflows/build.yml

```diff
@@ -11,13 +14,24 @@ env:
   IMAGE_NAME: ${{ github.repository }}
 
 jobs:
+  pull_request_lint: 
+    runs-on: ubuntu-latest
+
+    steps: 
+      - name: "Validate label"
+        if: | 
```

Comment:
Use job condition

Reply:
if we use the job condition, then if it is not met, the task will be skipped and the tasks that depend on this check will be skipped too, and we will get the status of done in PR. If we use such a check, then in case of successful passing we just skip a step inside the action, and in case of missing labels we exit with status 1, which marks the execution of the workflow with an error